src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt

```diff
@@ -261,17 +264,28 @@ class ComCaseloadService(
     }
   }
 
-  private fun buildVaryCaseload(managedOffenders: List<ManagedCase>): List<ManagedCase> = managedOffenders.filter { offender ->
-    offender.licences.any { licence ->
-      licence.licenceStatus in listOf(
-        ACTIVE,
-        VARIATION_IN_PROGRESS,
-        VARIATION_SUBMITTED,
-        VARIATION_APPROVED,
-        VARIATION_REJECTED,
-      ) ||
-        licence.isReviewNeeded
-    }
+  private fun getComDetails(casesToLicences: Map<ManagedOffenderCrn, LicenceSummary>): Map<String?, ProbationPractitioner> {
+    val comUsernames = casesToLicences.mapNotNull { (_, licence) -> licence.comUsername }.distinct()
+    val coms = deliusApiClient.getStaffDetailsByUsername(comUsernames)
+    return casesToLicences.map { (case, licence) ->
+      val com = coms.find { c -> licence.comUsername?.lowercase() == c.username?.lowercase() }
+      if (com != null) {
+        case.crn to ProbationPractitioner(
+          com.code,
+          name = com.name?.fullName(),
+          staffUsername = com.username,
+        )
+      } else {
+        if (case.staff == null || case.staff.unallocated == true) {
+          case.crn to ProbationPractitioner()
+        } else {
+          case.crn to ProbationPractitioner(
+            staffCode = case.staff.code,
+            name = case.staff.name?.fullName(),
+          )
+        }
+      }
+    }.toMap()
```

Comment:
Logic for this is taken from [here](https://github.com/ministryofjustice/create-and-vary-a-licence-api/blob/main/src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt#L236-L262). This still exists to support the COM create caseload, as I did not want to attempt to refactor all caseloads in one PR!